hypertrace-core-graphql-context/src/main/java/org/hypertrace/core/graphql/context/GraphQlRequestContextBuilder.java

```diff
@@ -3,9 +3,12 @@
 import graphql.kickstart.servlet.context.GraphQLServletContextBuilder;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.hypertrace.core.grpcutils.context.RequestContext;
 
 public interface GraphQlRequestContextBuilder extends GraphQLServletContextBuilder {
   @Override
   GraphQlRequestContext build(
       HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse);
+
+  RequestContext build(final HttpServletRequest httpServletRequest);
```

Comment:
GrpcContextBuilder already gives us this functionality. Can we compose them rather than duplicating?